### PR TITLE
feat: phone number verification

### PR DESCRIPTION
**Pull request by JudeDavis1**
This PR includes a Redux migration for registration and a primitive phone number verification.

### CODE REVIEW

File: backend/.env.example [hunk 1]
```diff
@@ -1,2 +1,4 @@
 DB_URL="host=db port=5432 user=postgres dbname=cms-db sslmode=disable password=postgres"
 CORS_ALLOW_ORIGIN="http://localhost:3000"
+JWT_SECRET="HI LOL"
```
Comment (j-koziel) [inline backend/.env.example:3]: hello 👋

File: packages/frontend/app/register/components/get-weekly-meetings.tsx [hunk 2]
```diff
@@ -1,22 +1,22 @@
...
 
-import { fetchLocalMeetings } from "@/lib/find-meetings/fetch-meetings";
-import { Congregation } from "@/lib/types/congregation";
 import React from "react";
 import { WeeklyMeetingsList } from "./weekly-meetings-list";
 import { Button } from "@nextui-org/button";
 import { MoveRight } from "lucide-react";
 import { createCongregation } from "@/lib/find-meetings/create-congregation";
 import { useRouter } from "next/navigation";
+import { fetchLocalMeetingsThunk } from "@/lib/stores/local-meetings";
```
Comment (j-koziel) [inline packages/frontend/app/register/components/get-weekly-meetings.tsx:9]: Thunk???

**Review by j-koziel — COMMENTED** [review]
LGTM 👍

File: packages/frontend/app/register/components/get-weekly-meetings.tsx (hunk 2 above)
Comment (JudeDavis1) [reply packages/frontend/app/register/components/get-weekly-meetings.tsx:9]: A thunk is a CS concept, basically a routine that is delayed, then its result is injected into where it's needed. Same principle here. 

Weird name ik.

**Pull request merged**